File: src/main/java/org/alham/alhamfirst/service/orchestrator/ai/AIService.java
package org.alham.alhamfirst.service.orchestrator.ai;


import reactor.core.publisher.Flux;

import java.util.Map;

public interface AIService {

    public String getAnswer(String question);

    public Flux<String> chat(String question);

    public Map<String ,Double> getStat(String statJson);

}

File: src/main/java/org/alham/alhamfirst/service/orchestrator/ai/AIService.kt
package org.alham.alhamfirst.service.orchestrator.ai;

import reactor.core.publisher.Flux;

interface AIService {

    fun getAnswer(question: String): String

    fun chat(question: String): Flux<String>

    fun getStat(statJson: String): MutableMap<String ,Double>

}
